CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# Default owners for everything in the repo, unless a later match takes precedence.
+* @grafana/loki-team
+
+# Documentation.
+/docs/ @grafana/docs-logs @grafana/loki-team
```

Comment:
There is an error here.

Reply:
I think it complains about team `@grafana/docs-logs` team being private, but that's fine.